src/snowflake/snowpark/mock/plan.py

```diff
@@ -622,14 +628,25 @@ def outer_join(base_df):
     )
 
 
-def describe(plan: MockExecutionPlan):
+def describe(plan: MockExecutionPlan) -> List[Attribute]:
     result = execute_mock_plan(plan)
-    return [
-        Attribute(
-            result[c].name, result[c].sf_type.datatype, result[c].sf_type.nullable
-        )
-        for c in result.columns
-    ]
+    ret = []
+    for c in result.columns:
+        if isinstance(result[c].sf_type.datatype, NullType):
+            ret.append(
+                Attribute(
+                    result[c].name if result[c].name else "NULL", StringType(), True
```

Comment:
Just curious: Do we support multiple columns of the same name `NULL` from `describe`?

Reply:
yes, you will get two columns of name `NULL` by the following scripts:
```python
    df2 = session.create_dataframe([1, 2]).to_df("a").select(lit(None), lit(None))
    assert len(df2.schema.fields) == 2
    assert df2.schema.fields[0].datatype == StringType()
```